index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/e32d72064d.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>The Complete 2022 Web Development Bootcamp</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico?v=2" />
+  </head>
+  <body>
+    <!-- MOBILE-POPUP -->
+    <div class="mobile-menu-background">
+      <i class="fa-solid fa-x close-menu"></i>
+      <ul>
+        <li><a class="close-menu-item1" href="#home">Home</a></li>
+        <li><a class="close-menu-item2" href="#about">About</a></li>
+        <li><a class="close-menu-item3" href="#featured">Featured</a></li>
+      </ul>
+    </div>
+
+    <!-- NAVIGATION BAR -->
+    <nav>
+      <i class="fa-solid fa-bars fa-2xl burger-menu"></i>
+      <div class="hidden">
+        <a href="#home"><i class="fa-solid fa-laptop-code fa-2xl logo"></i></a>
+        <ul>
+          <li><a class="menu-item1" href="#home">Home</a></li>
+          <li><a class="menu-item2" href="#about">About</a></li>
+          <li><a class="menu-item3" href="#featured">Featured</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
- [ ] The navigation bar is currently missing the top section, as pictured in the template (The one with the social icons), please make sure to add it.

![](https://i.imgur.com/JStFswR.png)

Reply:
Thanks, @ReshyResh for suggesting these factors. I will fix these right away 😊